conf/configuration.go

```diff
@@ -300,6 +301,7 @@ func init() {
 	viper.SetDefault("enableartworkprecache", true)
 	viper.SetDefault("autoimportplaylists", true)
 	viper.SetDefault("playlistspath", consts.DefaultPlaylistsPath)
+	viper.SetDefault("smartPlaylistRefreshTimeout", 5*time.Second)
```

Comment:
Hey, thanks for this. Can you make the value all lowercase, like `smartplaylistrefreshtimeout`? I think Viper has an issue with this.

Reply:
Oops my bad - should have picked that up from the other items. All fixed!